.github/workflows/workflow.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         ports:
           - 5672:5672
 
+      redis:
+        image: redis:6-alpine
+        ports:
+          - 6379:6379
```

Comment:
Now I am curious as why redis was not needed before :thinking:

Reply:
exactly

Reply:
that's what i'm looking for since yesterday

Reply:
Indeed, that's weird.
For the record, we added a Redis container inside unit tests in Kirin: https://github.com/CanalTP/kirin/blob/72ae5d7a65c79bcca4ae9fbdf850b59d5ed353eb/tests/docker_wrapper.py#L243
It seems a little better to avoid forcing redis availability on platform that's running test... But it probably depends on what requires redis.

Reply:
It's actually due to Celery 4.1.0. After comparing the behaviour "before/after", celery does now check the status of Redis when a task is sent(https://github.com/CanalTP/navitia/blob/dev/source/tyr/tyr/resources.py#L2095), which was not the case.

Reply:
:facepalm: What a mess... Are you sure celery is correctly setup?
We may have the same silent issue on kirin (as tests provide redis).

But it means you are actually testing celery tasks? (too complicated to be worth it on Kirin)

Reply:
Redis_result_backend is used as an call-back object when a task is sent (the redis-connection is sent to the worker, too), which was not the case